INSS.EIIR.DataAccess.Tests/INSS.EIIR.DataAccess.Tests.csproj

```diff
@@ -7,11 +7,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Remove="appsettings.json" />
+    <None Remove="appsettings-template.json" />
```

Comment:
could we try  deleting the ItemGroup section with  <None Remove="appsettings-template.json" />

Reply:
The change re-instates what was changed in the  commit cfba0fe, additionally this change was made by the IDE, I'd recommend it remain.